slider/beatmap.py

```diff
@@ -467,6 +469,7 @@ def __init__(self,
         self.repeat = repeat
         self.length = length
         self.ticks = ticks
+        self.tick_positions = tick_positions
```

Comment:
Thank you so much for spotting this! It's been causing one of my other projects to fail because my test is a DT play. I'll have a look at it.